packages/core/src/components/buttons/buttons.module.scss

```diff
@@ -9,54 +9,47 @@
 /* 'primary' | 'secondary' | 'link' | 'outlined'; */
 
 .appearance-primary {
-	color: white;
-	background: $colors-primary-a2 !important;
+	color: $white;
 	border: none;
 	border-radius: 8px;
 
 	&:hover {
-		background: $colors-accents-2 !important;
+		background: $colors-accents-2;
 	}
 
 	&:focus {
-		background: $colors-warning-1!important;
+		background: $colors-warning-1;
 		color: $colors-neutral-8;
 		box-shadow: 0px 2px 0px $colors-neutral-8;
+		outline: none;
 	}
-}
-
-.appearance-primary:disabled {
-	cursor: not-allowed !important;
-	background: $colors-primary-a1;
 
-	&:hover {
-		background: $colors-neutral-6 !important;
+	&:disabled {
+		cursor: not-allowed !important;
+		background: $colors-neutral-6;
+	
+		&:hover {
+			background: $colors-neutral-6 !important;
+		}
 	}
 }
 
 .appearance-secondary {
-	color: white;
-	background: $colors-primary-3;
+	color: $white;
+	background: $colors-primary-3!important;
 	border: none;
 	border-radius: 8px;
 
-	&:hover {
-		background: $colors-neutral-a1 !important;
-	}
-
 	&:focus {
 		background: $colors-warning-1!important;
 		color: $colors-neutral-8;
 		box-shadow: 0px 2px 0px $colors-neutral-8;
+		outline: none;
 	}
-}
 
-.appearance-secondary:disabled {
-	cursor: not-allowed !important;
-	background: $colors-primary-1;
-
-	&:hover {
-		background: $colors-neutral-6 !important;
+	&:disabled {
+		cursor: not-allowed !important;
+		background: $colors-primary-1!important;	
```

Comment:
probably need to add the `disabled->hover` state to keep the `background: $colors-primary-1`, currently is changing

Reply:
Fixed in PR #281